### ISSUE TITLE
Diffusion prior training trial run

### ISSUE THREAD

**Issue by xiankgx** [lucidrains/DALLE2-pytorch#86]
All l2norm related settings in DiffusionPrior() set to False as in the defaults, `image_embed_scale `set to 1.0 to disable any scaling. CLIP image embeddings are from OpenAI's `ViT-B/32` without any l2norm, dataset is cc3m. @lucidrains What do you think of the results? Seems to be working?

L1 training loss
![image](https://user-images.githubusercontent.com/4113258/167965166-620255bc-42cd-48b1-a6b8-efea61aeb4aa.png)

Training batch CLIP image embed to sampled CLIP image embed (from training batch text) L2 loss
![image](https://user-images.githubusercontent.com/4113258/167965194-0561f309-501d-42a6-9949-a144dd8c851f.png)

Training batch CLIP text embed to sampled CLIP image embed (from training batch text) softmax accuracy
![image](https://user-images.githubusercontent.com/4113258/167965251-07741858-beb0-43d3-84a6-c766536bd373.png)

Training batch CLIP text embed to sampled CLIP image embed (from training batch text) cosine similarity
![image](https://user-images.githubusercontent.com/4113258/167965222-fa7eca53-86f4-4791-a4c4-030df6d670a1.png)

**Comment by egeozsoy** [comment]
Any recommendations on batch size?

**Comment by lucidrains** [comment]
@egeozsoy 
<img width="656" alt="Screen Shot 2022-05-13 at 3 04 15 PM" src="https://user-images.githubusercontent.com/108653/168395185-152da082-e632-4445-a3dd-9b3345520eb5.png">

table 3 in appendix says they used 4096. but evidently people are training it successfully on much smaller batch sizes
